app.js

```diff
@@ -2,9 +2,20 @@ const express = require("express");
 const app = express();
 
 const { getTopics } = require("./controllers/topics_controllers");
+const { getArticleById } = require("./controllers/articles_controllers");
+const {
+  handlePSQLErrors,
+  handleCustomErrors,
+  handleInternalErrors,
+} = require("./controllers/error_controllers.js");
 
 app.use(express.json());

```

Comment:
do we need this line yet? 👀

Reply:
No, but last week onece I was stuck on kata for around 1 hour because i havent had it, now i will never forget to write it even if i'm not using it at the moment